keras_nlp/models/opt/opt_causal_lm_preprocessor.py

```diff
@@ -165,6 +166,10 @@ def generate_postprocess(
         back to a string.
         """
         token_ids, padding_mask = x["token_ids"], x["padding_mask"]
+        if not isinstance(token_ids, tf.Tensor):
```

Comment:
Is the reason for this check that we may be in a tf.data graph, in which case we want to keep using Tensors?

If not, I'm curious if we can just always `convert_to_numpy`